README.md

```diff
@@ -4,6 +4,15 @@
 
   [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/oadp-operator)](https://goreportcard.com/report/github.com/openshift/oadp-operator) [![codecov](https://codecov.io/gh/openshift/oadp-operator/branch/master/graph/badge.svg?token=qLM0hAzjpD)](https://codecov.io/gh/openshift/oadp-operator) [![License](https://img.shields.io/:license-apache-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html) [![Go Reference](https://pkg.go.dev/badge/github.com/openshift/oadp-operator.svg)](https://pkg.go.dev/github.com/openshift/oadp-operator)
 
+OADP repositories images job
+[![OADP repository](https://prow.ci.openshift.org/badge.svg?jobs=branch-ci-openshift-oadp-operator-master-images)](https://prow.ci.openshift.org/job-history/gs/test-platform-results/logs/branch-ci-openshift-oadp-operator-master-images)
```

Comment:
Can we add some text per badge to help identify the badge per repo ?

Reply:
I do not think so

This displays the image in the badge URL; if the URL does not exist, the image would not be generated and alt text would be shown (in this case `OADP repository`)

Wat we could do is have a section for each repo, instead of one for all images, for better separate them

Reply:
yes a table would be great

Reply:
will do follow up then!